### DISCUSSION TITLE
Export from Zettlr to PDF

### DISCUSSION THREAD

**Discussion by riccardo1974** [Zettlr/Zettlr#4770] [General]
Hi to all, I'm following this guide to modify my .tex template:
https://docs.zettlr.com/en/academic/custom-templates/

At section "Activate your template" I can't see on Preferences-->Asset Manager the tab PDF (I mean the Default PDF), is it correct to choose XeLaTeX PDF? Have I to modify it?
Ric

**Comment by nathanlesage** [comment]
Yes, this one is now called "XeLaTeX PDF" and yes, you'd need to follow the instructions to modify it. You can also rename that profile, which will essentially create a copy which you can then modify in order to keep the "original" profile.

**Comment by riccardo1974** [comment]
Thanks a lot, works perfectly!

Il giorno mar 7 nov 2023 alle ore 18:49 Hendrik Erz <
***@***.***> ha scritto:

> Yes, this one is now called "XeLaTeX PDF" and yes, you'd need to follow
> the instructions to modify it. You can also rename that profile, which will
> essentially create a copy which you can then modify in order to keep the
> "original" profile.


EDIT:

I make change in my template but PDF generated is always the same, no change. Or also no PDF are generated, non error message, simply no generated PDF. Instead if I put the same latex code in TexWorks and compile I can see the change made, any help? where am I wrong in Zettlr?

**Comment by riccardo1974** [comment]
UPDATE: I write down this simple LaTeX code into my default-template-modified.tex to make all my chapters titles in red color

```
\documentclass[statementpaper,11pt,openright,twoside,extrafontsizes,draft]{memoir} 
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage[italian]{babel}
\usepackage{titlesec}

\usepackage{xcolor}
\usepackage{sectsty}
\chapterfont{\color{red}}

\begin{document}

$body$

\end{document}

```

But when I try to export using my PDF profile (linked to default-template-modified.tex) nothing happens, no error and no generated PDF. If I delete this code:
`\usepackage{xcolor}
\usepackage{sectsty}
\chapterfont{\color{red}}`

PDF is generated but title of the chapter (# Title in text file .md) is black obviously.

In the file .md with my text body I want to export in PDF how Zettlr can understand that # Title markdown have to be converted in command \chapter{Title} as in Latex? here is the error I suppose. 

Infact if I copy the first code above in TeXworks it works perfectly.

Can someone help?
